### PR TITLE
doc/rados: rewrite mclock docs (2 of 2)

### PR DESCRIPTION
**Pull request by zdover23**
This is my second attempt to rewrite the
second half of the mclock docs. The first attempt
is enshrined in https://github.com/ceph/ceph/pull/40571,
in which I got cute with git and got burned.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by sseshasa — APPROVED** [review]
LGTM

File: doc/rados/configuration/mclock-config-ref.rst [hunk 1]
```diff
@@ -205,43 +205,48 @@ correct bluestore throttle values if desired,
...
+option ``osd_mclock_max_capacity_iops_[hdd, ssd]`` can be set by specifying it
+in either the **[global]** section or in a specific OSD section (**[osd.x]** of
+your Ceph configuration file).
 
-Alternatively, the following commands may be used,
+Alternatively, commands of the following form may be used:
 
-``$ ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd]
-<value>``
+  .. prompt:: bash $
```
Comment (neha-ojha) [inline doc/rados/configuration/mclock-config-ref.rst:229]: We use the following everywhere in our documentation, should we use the same in this doc too, just to be consistent?

```suggestion
  .. prompt:: bash #
```

File: doc/rados/configuration/mclock-config-ref.rst [hunk 2]
```diff
@@ -205,43 +205,48 @@ correct bluestore throttle values if desired,
...
+    ceph config set osd osd_mclock_max_capacity_iops_ssd 25000
 
+To set the capacity for a specific OSD (for example "osd.0") whose underlying
+device type is HDD, use a command like this:
 
-Specify mClock Profile to Enable
----------------------------------
+  .. prompt:: bash $
+
+    $ ceph config set osd.0 osd_mclock_max_capacity_iops_hdd 350
```
Comment (neha-ojha) [inline doc/rados/configuration/mclock-config-ref.rst:245]: extra $

```suggestion
    ceph config set osd.0 osd_mclock_max_capacity_iops_hdd 350
```

Comment (zdover23) [reply doc/rados/configuration/mclock-config-ref.rst:245]: Aw. I thought I got all of these.

**Comment by zdover23** [comment]
jenkins test docs

**Comment by neha-ojha** [comment]
jenkins test docs

**Review by neha-ojha — APPROVED** [review]

**Pull request merged**